telemetry/otel-collector-config.yaml

```diff
@@ -17,6 +17,8 @@ exporters:
 
   prometheus:
     endpoint: '0.0.0.0:8889'
+    resource_to_telemetry_conversion:
+      enabled: true
```

Comment:
I also had to add this, as the local telemetry setup was dropping metric annotations by default, which confused me for a while.